pkg/utils/validator/clusterresourceplacement_test.go

```diff
@@ -199,9 +274,10 @@ func TestValidateClusterResourcePlacementAlpha(t *testing.T) {
 
```

Comment:
you can revert the v1alpha1 changes and you don't need to enhance the deprecating api